numpy/lib/function_base.py

```diff
@@ -2325,6 +2325,10 @@ def cov(m, y=None, rowvar=True, bias=False, ddof=None, fweights=None,
         weights can be used to assign probabilities to observation vectors.
 
         .. versionadded:: 1.10
+    dtype : data-type, optional
+        Data-type of the result. By default, the return data-type is np.float64.
```

Comment:
```suggestion
        Data-type of the result. By default, the return data-type is `numpy.float64`.
```
which will produce a link